src/app/meditation/testing/fake-meditation.service.ts

```diff
@@ -0,0 +1,158 @@
+import { TestHelper } from '../../../testing/test.helper';
+import { Observable } from 'rxjs/Rx';
+
+export class FakeMeditationService {
+
+  public getRecent() {
+    return TestHelper.fakeResponse([
+      {
+        '_id': '151',
+        'updatedAt': '2017-06-10T06:20:13.856Z',
+        'sitting': 40,
+        'walking': 0,
+        'createdAt': '2017-06-10T06:16:33.905Z',
+        'end': '2017-06-10T06:56:33.905Z',
+        'user': {
+          '_id': '5',
+          'gravatarHash': '4de3df4fa010cd20c1c226fda92b3b83',
```

Comment:
Those records contain user data from the website. We are not allowed to expose these in any way online. Please anonymize all names and change the gravatar hashes to `d8e8fca2dc0f896fd7cb4cb0031ba249` (or any other arbitrary md5 hash).